### ISSUE TITLE
Valid security page returns 404 triggering action 

### ISSUE THREAD

**Issue by 0ddInput** [disclose/diodb#291]
Validate links github action gets triggered because of the following URL:
https://saymine.com/vulnerability-disclosure

**Comment by sickcodes** [comment]
I emailed the website owner out of courtesy

**Comment by prodigysml** [comment]
If we don't hear back within the next two weeks, we'll prolly just hardcode a skip in the action (yes, smelly, but best we can do without the action going nuts).

**Comment by yesnet0** [comment]
Checking in on this one @prodigysml

**Comment by sickcodes** [comment]
They emailed me back but never fixed it, hard skip broken site?

**Comment by prodigysml** [comment]
Yeah, we will need a hardcoded skip for it. Happy to put it in if we agree that's the way we want move forward
